mne/viz/_3d.py

```diff
@@ -1178,7 +1180,7 @@ def _plot_sensors(renderer, info, to_cf_t, picks, meg, eeg, fnirs,
                   warn_meg, head_surf, units, sensor_opacity=0.8,
                   orient_glyphs=False, scale_by_distance=False,
                   project_points=False, surf=None, check_inside=None,
-                  nearest=None):
+                  nearest=None, sensor_colors: np.array = None):
```

Comment:
let's not add type hints for just one param of one function. If we're going to start annotating we should first agree on a policy and approach. That approach might end up being "add them one param at a time as they get touched" but so far no concrete decision / plan is in place so let's not force our hand here.